data/json/effects_on_condition/mutation_eocs/mutation_effect_eocs.json

```
@@ -685,5 +685,17 @@
     "//": "will remove ink_gland_ink effect when the mutation is lost",
     "condition": { "and": [ { "u_has_effect": "ink_gland_ink" }, { "compare_string": [ "INK_GLANDS", { "context_val": "trait" } ] } ] },
     "effect": [ { "u_lose_effect": "ink_gland_ink" } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_insect_wings_shutoff",
+    "recurrence": [ "1 seconds", "1 seconds" ],
+    "condition": {
+      "and": [ { "compare_num": [ { "u_val": "stamina" }, "<", { "const": 3000 } ] }, { "u_has_trait": "WINGS_INSECT_active" } ]
+    },
```

how expensive it is to run EoC every second? anyway `deactivate_condition` should cover us here

```suggestion
    "recurrence": "1 seconds",
    "condition": { "math": [ "u_val('stamina') * 2", "<", "3000" ] },
    "deactivate_condition": { "not": { "u_has_trait": "WINGS_INSECT_active" } },
```